app/views/adjustments/index.html.erb

```diff
@@ -50,7 +54,7 @@
       <div class="row">
         <div class="col-xs-12">
           <div class="box-body table-responsive no-padding">
-            <table class="table table-hover striped">
+            <table class="table table-hover striped records">
```

Comment:
the "records" class is for a test hook

Reply:
I like using `data`-attributes for hooking into tests.